docs/source-app/examples/github_repo_runner/app.py

```diff
@@ -198,7 +198,13 @@ def page_1__create_new_run(state):
         "Enter a Github Repo URL", value="https://github.com/Lightning-AI/lightning-quick-start.git"
     )
 
-    default_script_args = "--trainer.max_epochs=5 --trainer.limit_train_batches=4 --trainer.limit_val_batches=4 --trainer.callbacks=ModelCheckpoint --trainer.callbacks.monitor=val_acc"
+    default_script_args = (
```

Comment:
The re-formatting is breaking the docs. Now, we need to re-aligned the cut for the tutorials. Can you re-align them ?

Reply:
well, on one side you say you format to have short line, but this was too long